packages/attestation-service/migrations/20200506234935-attestation-v2.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const transaction = await queryInterface.sequelize.transaction()
+
+    try {
+      await queryInterface.removeColumn('Attestations', 'phoneNumber')
+
+      await queryInterface.addColumn('Attestations', 'identifier', {
+        type: Sequelize.STRING,
+      })
+
+      await queryInterface.removeIndex(
```

Comment:
Did removing the index after removing the column work? I would have thought that one needed to remove the index first

Reply:
i did work... but that's a good point.  perhaps i should just inverse these to be safe